packages/event-emitter/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@react-native/event-emitter",
+  "version": "1.0.0",
```

Comment:
Let's not use `1.0.0` here as it will force us to rename/deprecate this package afterwards. This should be aligned with the RN version so ideally `0.71.0` for now.

(also cc @kelset as he's taking care of this).

Reply:
Thanks! I will include this fix with the rebase after https://github.com/facebook/react-native/pull/34423 is merged.